sof_wrapper/auth/views.py

```diff
@@ -1,24 +1,27 @@
 from flask import Blueprint, current_app, redirect, request, url_for, session
-from urllib.parse import urlencode
-import base64
 import requests
 
 from sof_wrapper.extensions import oauth
+from sof_wrapper.auth.helpers import extract_payload
 
 
 blueprint = Blueprint('auth', __name__, url_prefix='/auth')
 
 
 def debugging_compliance_fix(session):
     def _fix(response):
-        current_app.logger.info('access_token request url: %s', response.request.url)
-        current_app.logger.info('access_token request headers: %s', response.request.headers)
-        current_app.logger.info('access_token request body: %s', response.request.body)
-
-
-        current_app.logger.info('access_token response: %s', response)
-        current_app.logger.info('access_token response.status_code: %s', response.status_code)
-        current_app.logger.info('access_token response.content: %s', response.content)
+        current_app.logger.debug(
+            'access_token request url: %s', response.request.url)
+        current_app.logger.debug(
+            'access_token request headers: %s', response.request.headers)
+        current_app.logger.debug(
+            'access_token request body: %s', response.request.body)
+
+        current_app.logger.debug('access_token response: %s', response)
+        current_app.logger.debug(
+            'access_token response.status_code: %s', response.status_code)
+        current_app.logger.debug(
+            'access_token response.content: %s', response.content)
```

Comment:
This is a lot less consistent/readable. Would prefer to use 100 characters across the board, but don't care enough to switch this back :)